app/Platform/Actions/ComputeGameSortTitleAction.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Actions;
+
+class ComputeGameSortTitleAction
+{
+    public function execute(string $gameTitle): string
+    {
+        $sortTitle = $this->replaceRomanNumerals($gameTitle);
+        $sortTitle = $this->removeArticles($sortTitle);
+        $sortTitle = mb_strtolower($sortTitle);
+        $sortTitle = $this->fixTagTildes($sortTitle);
+
+        return $sortTitle;
+    }
+
+    /**
+     * Replace Roman numerals with their corresponding padded numeric
+     * equivalents (eg: "IV" -> "04").
+     */
+    private function replaceRomanNumerals(string $title): string
+    {
+        $romanNumerals = [
+            'I' => 1, 'II' => 2, 'III' => 3, 'IV' => 4, 'V' => 5,
```

Comment:
Need to be careful with single letters (`I` in particular):
```
| I Have No Mouth, and I Must Scream | 01 have no mouth, and 01 must scream |
| Rugrats: I Gotta Go Party          | rugrats: 01 gotta go party           |
| I'm Sorry                          | 01'm sorry                           |
| ~Hack~ V I T A L I T Y             | ~hack 05 01 t a l 01 t y             |
| ~Hack~ Dragoon X Omega             | ~hack dragoon 10 omega               |
| ~Hack~ Pokemon - X and Y           | ~hack pokemon - 10 and y             |
```
It seems like you should ignore any single letter match followed by an apostrophe, or another word. Valid conversions are at the end of a string, or followed by non-apostrophe punctuation (or whitespace then punctuation) like `:`, `-`, `&`, or `(`.

Reply:
Great feedback, this should now be addressed in latest. I've also refactored how the test cases execute to make it easier to add/update test cases.

After some consideration, I think we may want to go even further on a few things with the algorithm.

1. Arabic Numbers in Titles
I think it may be desirable to automatically pad all Arabic numbers in titles with leading zeros to ensure proper sorting. Otherwise, "Mega Man 10" might automatically be sorted before "Mega Man 2".

2. Special Characters & Accents
It may be desirable to automatically normalize "Pokémon" to "Pokemon" (no accent). In other words, normalize accented chars to their ASCII equivalents.

Reply:
Agreed. For searches, I believe Laravel Scout's Meilisearch driver (for which there are already env vars configured for) automatically accounts for this.